Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -170,9 +170,12 @@ void CookieStore::getAll(CookieStoreGetOptions&& options, Ref<DeferredPromise>&&
             return;
         }
 
-        promise->resolve<IDLSequence<IDLDictionary<CookieListItem>>>(cookies->map([](auto& cookie) {
```

Comment:
We should try to keep using `map()`, can you try the following?
```
promise->resolve<IDLSequence<IDLDictionary<CookieListItem>>>(WTF::map(WTFMove(cookies), [](auto&& cookie) {
    return CookieListItem { WTFMove(cookie) };
}));
```

Reply:
I was trying different ways to use map (I just tried the one above too) but it seems like the lambda that is passed to map must be const and so we aren't allowed to WTFMove the arguments passed in? The above method resulted in some errors too.

Reply:
Pretty sure we can get it to work. We have plenty of existing examples in our code base:
```
LayerAncestorClippingStack::LayerAncestorClippingStack(Vector<CompositedClipData>&& clipDataStack)
    : m_stack(WTF::map(WTFMove(clipDataStack), [](auto&& clipDataEntry) { return ClippingStackEntry { WTFMove(clipDataEntry), 0, nullptr, nullptr }; }))
{
}
``` 

for example.

Reply:
Ah alright, I figured out the issue. It should work now.